File: crates/harn-hostlib/tests/fixtures/ast/swift/source.swift
protocol Speaker {
    func speak() -> String
}

struct Point {
    var x: Int
    var y: Int
}

enum Color {
    case red
    case green
    case blue
}

class Greeter: Speaker {
    let name: String

    init(name: String) {
        self.name = name
    }

    func speak() -> String {
        return "hello \(name)"
    }
}

func shout(_ s: String) -> String {
    return s.uppercased()
}

File: crates/harn-hostlib/tests/fixtures/ast/java/source.java
public class Greeter {
    private final String name;

    public Greeter(String name) {
        this.name = name;
    }

    public String greet() {
        return "hello " + name;
    }
}

interface Speaker {
    String speak();
}

enum Color {
    RED,
    GREEN,
    BLUE
}
